payments/price_feed.py

```diff
@@ -1,16 +1,27 @@
 import requests
-
 import config
 
 
 def get_price(currency):
-    price_feed = "https://api.coindesk.com/v1/bpi/currentprice.json"
+
+    # Define some currency_provider-specific settings 
+    if config.currency_provider == "COINDESK":
+        price_feed  = "https://api.coindesk.com/v1/bpi/currentprice.json"  # url to get the pricefeed 
+        result_root = "bpi"               # name of the outer json tag
+        value_attribute   = "rate_float"  # attribute in which the float fx is located in json reposnse
+        ticker      = currency.upper()                     
```

Comment:
Maybe make these variables returned from a function or from a dict. This will make the function smaller and stop you from repeating the inline comments multiple times